### PR TITLE
home page update - add events api ref.

### PR DESCRIPTION
**Pull request by meganelepalud-akeneo**
Screenshot 2021-01-20 at 10 33 07

### CODE REVIEW

**Review by pierrejolly — APPROVED** [review]

File: src/index.handlebars [hunk 1]
```diff
@@ -172,14 +172,15 @@
                </div>
             </div>
             <div class="col-xs-12 col-sm-offset-3 col-sm-6 col-md-offset-0 col-md-4">
-                <h2 class="text-center">Our REST API reference</h2>
+                <h2 class="text-center rest">Our references</h2>
```
Comment (pierrejolly) [inline src/index.handlebars:175]: Why do you add a `rest` class here ?

Comment (meganelepalud-akeneo) [reply src/index.handlebars:175]: We now have 2 colors for the API references: one for the REST API and another one for the Events API.

Comment (meganelepalud-akeneo) [reply src/index.handlebars:175]: I forgot to delete it after some CSS tests.

Comment (meganelepalud-akeneo) [inline src/index.handlebars:175]:
```suggestion
                <h2 class="text-center">Our references</h2>
```

**Review by willy-ahva — APPROVED** [review]

**Pull request merged**